libs/user-monitoring/src/lib/user-monitoring.ts

```diff
@@ -0,0 +1,33 @@
+import { datadogRum } from '@datadog/browser-rum'
+
+import { maskNationalId } from '@island.is/shared/pii'
+
+interface DdRumInitParams {
+  env: string
+  version: string
+  service: string
+}
+
+const initDdRum = (params: DdRumInitParams) => {
+  datadogRum.init({
+    applicationId: '665a387d-16c4-4c86-88c8-e059d8120021',
+    clientToken: 'pub7a898a272110329d3a2e73e09e263f5b',
```

Comment:
Do we want to hard code this here?

Reply:
No probably not, I moved this to environment variables.